src/server/chunk_server/chunk_server_file_service_impl.cc

```diff
@@ -0,0 +1,50 @@
+#include "src/server/chunk_server/chunk_server_file_service_impl.h"
+
+#include "grpcpp/grpcpp.h"
+#include "src/protos/grpc/chunk_server_file_service.grpc.pb.h"
+
+using grpc::ServerContext;
+using protos::grpc::AdvanceFileChunkVersionReply;
+using protos::grpc::AdvanceFileChunkVersionRequest;
+using protos::grpc::InitFileChunkReply;
+using protos::grpc::InitFileChunkRequest;
+using protos::grpc::ReadFileChunkReply;
+using protos::grpc::ReadFileChunkRequest;
+using protos::grpc::WriteFileChunkReply;
+using protos::grpc::WriteFileChunkRequest;
+
+namespace gfs {
+namespace service {
+
+grpc::Status ChunkServerFileServiceImpl::InitFileChunk(
+    ServerContext* context, const InitFileChunkRequest* request,
+    InitFileChunkReply* reply) {
+  // TODO(someone): implement the GFS chunk server logic here
```

Comment:
I don't feel the need to be explicit here as the work kind of overlaps. Matthew owns the client API -- which is how a GFS client talks to servers to read/write the data -- but no one currently owns the implementations of any of the server side handling logic

Reply:
Mentioned this because this issue (https://github.com/Michael-Tu/cppGFS2.0/issues/11) shows that matthew owns File chunk manager, which essentially handles reading and writing chunks on the chunkserver. Okay, it's fine.